autogen/logger/logger_factory.py

```diff
@@ -14,5 +15,7 @@ def get_logger(logger_type: str = "sqlite", config: Optional[Dict[str, Any]] = N
 
         if logger_type == "sqlite":
             return SqliteLogger(config)
+        if logger_type == "agentops":
```

Comment:
When resolving the conflict, let's remove this line. When user needs to use it, instead of using `logger_type`, just pass in the `logger` object which is an AgentOpsLogger.

Reply:
yeah this approach fits well.

Reply:
this makes a lot more sense! now we can definitely have the logger on our side :)